hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -2415,4 +2434,17 @@
     <xs:simpleType name="wan-ack-type">
         <xs:union memberTypes="wan-ack-type-format-enum non-space-string"/>
     </xs:simpleType>
+
+    <xs:complexType name="hot-restart">
+        <xs:sequence>
+            <xs:element name="home-dir" type="xs:string" minOccurs="0" maxOccurs="1" default="hot-restart"/>
+        </xs:sequence>
+        <xs:attribute name="enabled" type="xs:string">
```

Comment:
@bilalyasar: because in Spring you can replace config values with variable expressions, like `${hotrestart.enabled}`. But you can't write that expression to a `xs:boolean` type. All elements & attributes in spring xml should be `xs:string`, if you ask me.